docker-compose.yml

```diff
@@ -7,10 +7,35 @@ services:
     restart: always
     ports:
       - "8888:3000"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.basic.frontend.rule=Host:explorer.raiden.network"
```

Comment:
Also the `basic` part isn't needed here. That is only necessary if there need to be multiple configurations per container.